generated/resources/transform/role/name_test.go

```diff
@@ -0,0 +1,96 @@
+package role
```

Comment:
It's not generated. I decided that since generating code is so new, all the tests should be hand-written so that we're really taking the time to make sure we're getting the quality we expect.

Reply:
:+1: